### ISSUE TITLE
Configuration of apt / yum repository

### ISSUE THREAD

**Issue by dhoppe** [n1tr0g/golja-influxdb#30]
Please make this optional. The user should be able to configure its own repository.

https://github.com/n1tr0g/golja-influxdb/blob/master/manifests/server/install.pp#L17

**Comment by n1tr0g** [comment]
Did you try to use the manage_install parameter to ignore the module installation or you want to have an way that inside this module you can provide custom repo's ?

**Comment by dhoppe** [comment]
I just need an additional parameter to enable/disable the include of class repo.pp. In some cases the user might be interested in using its local mirror instead of the public mirror of InfluxData.

**Comment by dannygoulder** [comment]
agreed, it would be great if the module could be configurable to simply install the influxdb package.  In my case I use Foreman to manage local repo mirrors (as the influx servers don't have access to the interwebs)

**Comment by jbiel** [comment]
+1 to make the upstream repo optional. I'd like to install from my own repository instead.

**Comment by danielpalstra** [comment]
At my current assignment we manage local repo mirrors. I've made pull request #38 to make repo management optional.
